Infrastructure/Services/PostcodesIoLocationFilterService.cs

```diff
@@ -48,8 +48,13 @@ public PostcodesIoLocationFilterService(HttpClient httpClient, IUnitOfWork unitO
         parameters.LocalAuthorityDistrictId = lad.Id;
         parameters.LocalAuthorityDistrictCode = lad.Code;
         parameters.LocalAuthorityDistrict = lad.Name;
-        var school = await _unitOfWork.SchoolRepository.FirstOrDefaultAsync(e => e.Postcode == parameters.Postcode);
-        parameters.Urn = school?.Urn;
+        var schools = await _unitOfWork.SchoolRepository.GetAllAsync(e => e.Postcode == parameters.Postcode && e.IsActive, "PhaseOfEducation");
```

Comment:
Since we're only retrieving data in this scenario, I suggest we disable tracking for this query. By default, tracking is enabled, but we can turn it off to optimize performance.

Reply:
Done